common/scripts/run_rush_change.py

```diff
@@ -16,9 +16,9 @@
 
 # Verifying with rush change requires the branch that is being merged into to be provided.  More details, https://rushjs.io/pages/commands/rush_change/.
 # With release/* being a potential target branch in addition to master, special case those branches.
-if targetBranch.find("refs/heads/release") != -1:
+if "refs/heads/release" in targetBranch:
     branchCmd = ["-b", targetBranch.replace("refs/heads/", "origin/")]
-elif targetBranch.find("release") != -1:
+elif "release" in targetBranch or targetBranch == srcBranch:
     # ADOps uses the branch name (i.e. 'release/2.8.0') for GH PR branch names instead of full refs.
```

Comment:
Could you add a comment here for our use case?

Reply:
new comment should be on line 23